variants/XMC4200/config/XMC4200_Distance2Go/pins_arduino.h

```diff
@@ -0,0 +1,226 @@
+/*
+  pins_arduino.h - Pin definition functions for Arduino
+  Part of Arduino - http://www.arduino.cc/
+
+  Copyright (c) 2007 David A. Mellis
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General
+  Public License along with this library; if not, write to the
+  Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+  Boston, MA  02111-1307  USA
+  
+  Copyright (c) 2019 Infineon Technologies AG
+  This file has been modified for the XMC microcontroller series.
+*/
+#ifndef PINS_ARDUINO_H_
+#define PINS_ARDUINO_H_
+
+//****************************************************************************
+// @Project Includes
+//****************************************************************************
+#include <XMC4200.h>
+
+//****************************************************************************
+// @Defines
+//****************************************************************************
+// XMC_BOARD for stringifying into serial or other text outputs/logs
+// Note the actual name XMC and number MUST have a character between 
+// to avoid issues with other defined macros e.g. XMC1100
+#define XMC_BOARD           XMC 4200 Distance2Go
+/* On board LED is ON when digital output is 0, LOW, False, OFF */
+#define XMC_LED_ON          0
+// Pins that are allocated or free to be used as Analog input
+#define NUM_ANALOG_INPUTS   6
+#define NUM_PWM             6
+#define NUM_LEDS            3
+#define NUM_INTERRUPT       2
+#define NUM_SERIAL          1
+#define NUM_TONE_PINS       8
+#define NUM_TASKS_VARIANT   16
```

Comment:
Considering device size and clock rate I would consider making this 16 or 32, these are software driven tasks from systick counts down to ms resolution